functions/.gitignore

```diff
@@ -0,0 +1,8 @@
+## Compiled JavaScript files
+**/*.js
```

Comment:
Just to verify, should we ignore .js files as well?

Reply:
This glob should handle everything including *.js